content/scripts/build.js

```diff
@@ -441,19 +441,26 @@ class Builder {
    * rendering all of this document's prerequisites, taking into
    * account that the prerequisites may have prerequisites and so on.
    */
-  async renderMacros(source, rawHtml, metadata, { cacheResult = false } = {}) {
-    const prerequisites = kumascript.getPrerequisites(rawHtml);
+  async renderMacros(
+    source,
+    rawHtml,
+    metadata,
+    fileInfo,
+    { cacheResult = false } = {}
+  ) {
     const mdn_url = buildMDNUrl(metadata.locale, metadata.slug);
     const uri = mdn_url.toLowerCase();
 
     // First, render all of the prerequisites of this document.
     let allPrerequisiteFlaws = [];
-    for (const prerequisite of prerequisites) {
+    for (const prerequisite of kumascript.getPrerequisites(rawHtml)) {
       const prerequisiteUri = this.cleanUri(prerequisite.uri);
       if (!this.allTitles.has(prerequisiteUri)) {
         allPrerequisiteFlaws.push(
-          prerequisite.createFlaw(
-            `${uri} transcludes ${prerequisiteUri}, which does not exist`
+          fileInfo.updateFlaw(
+            prerequisite.createFlaw(
+              `${uri} transcludes ${prerequisiteUri}, which does not exist`
```

Comment:
I still don't know what [transclusion](https://en.wikipedia.org/wiki/Transclusion) means but I'm slowly learning. 

But how do I see this? Do you know of a page which triggers this?
On http://localhost:3000/en-US/docs/Web/HTML/Element/input/range#_flaws I see `/en-US/docs/Web/API/HTMLInputElement/value does not exist`

Reply:
I had to look up `transclusion` as well, after I first heard it used by Will. 😄 Should we change `transcludes` to just `references`?

You can see this on http://localhost:3000/en-us/docs/web/fubar#_flaws. It's the first flaw of the 6 flaws shown.